### DISCUSSION TITLE
Modifying source code for WordPress Plugin

### DISCUSSION THREAD

**Discussion by aphn24** [bullhorn/career-portal#435] [Q&A]
I was able to modify the source code according to my need like implementing styles and other changes. But now I want to implement it in my WordPress site. And the Plugin code and the source code are different. What am I supposed to do now?

**Comment by charlesabarnes** [comment]
Hello, 

You need to [build](https://github.com/bullhorn/career-portal#building) the portal, then you can add the built files to the plugin directory.

**Reply by aphn24** [reply]
Thank you very much. It did work.
